docs/section11/Rule 11-17.md

```diff
@@ -0,0 +1,49 @@
+# Service_Water_Heating - Rule 11-17  
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE  
+
+**Rule ID:** 11-17  
+
+**Rule Description:** "All buildings that will have service water heating loads must include those loads in the simulation"  
+
+**Rule Assertion:** PASS/FAIL/UNDETERMINED  
+
+**Appendix G Section Reference:** Table G3.1 #1, proposed column, (a)  
+
+**Evaluation Context:** P_RMD - each building segment  
+**Data Lookup:**   
+**Function Call:**  
+- **get_SWH_bats_and_SWH_use**  
+- **get_building_segment_SWH_bat**  
+- **get_SWH_uses_associated_with_each_building_segment**  
+- **get_component_by_id**  
+
+## Applicability Checks:  
```

Comment:
Formatting of Applicability Checks is awkward. Suggest removing it since it seems like it is applicable to every project that has a building segment - which is always?

Reply:
I see your point, but I think this is one of the standard, required sections for RDSs